docs/configuration.md

```diff
@@ -10,7 +10,8 @@ Variable | Description
 ---|---
 `APP_ID` | The App ID assigned to your GitHub App. **Required** <p>_(Example: `1234`)_</p>
 `PRIVATE_KEY_PATH` | The path to the `.pem` file for your GitHub App. If not present, Probot will look for a file ending in `.pem` in your project's directory. **Required** if the `.pem` file is not in your project directory and `PRIVATE_KEY` is not set. <p>_(Example: `path/to/key.pem`)_</p>
-`PRIVATE_KEY` | The contents of the private key for your GitHub App. **Required** if the `.pem` file is not in your project directory and `PRIVATE_KEY_PATH` is not set.
+`PRIVATE_KEY` | The contents of the private key for your GitHub App. **Required** if the `.pem` file is not in your project directory and `PRIVATE_KEY_PATH` or `PRIVATE_KEY_BASE64` is not set.
```

Comment:
I'm wondering if maybe we should unbold the conditional "Required"s, to de-emphasize them?

Reply:
Yeh I was thinking the same thing. It's confusing to say they're all required. Thinking maybe we group them somehow so it's obvious you need to provide one of the three options.

Reply:
@hiimbex How about this?

Variable | Description
---|---
`APP_ID` | The App ID assigned to your GitHub App. **Required** <p>_(Example: `1234`)_</p>
**Private Key Options** | One of the following is **Required** if there is no `.pem` file in your project's root directory
`PRIVATE_KEY_PATH` | The path to the `.pem` file for your GitHub App. <p>_(Example: `path/to/key.pem`)_</p>
`PRIVATE_KEY` | The contents of the private key for your GitHub App. | 
`PRIVATE_KEY_BASE64` | In environments where you don't want to upload the private key and can't set a multiline environment variable, you can base64 encode the `.pem` file and set this variable.
**Webhook Options** | 
`WEBHOOK_PROXY_URL` | Allows your local development environment to receive GitHub webhook events. Go to https://smee.io/new to get started. <p>_(Example: `https://smee.io/your-custom-url`)_</p>
`WEBHOOK_SECRET` | The webhook secret used when creating a GitHub App. 'development' is used as a default, but the value in `.env` needs to match the value configured in your App settings on GitHub. Note: GitHub marks this value as optional, but for optimal security it's required for Probot apps. **Required** <p>_(Example: `development`)_</p>